tests/business-logic/Users.test.ts

```diff
@@ -0,0 +1,84 @@
+import UserEntity from "@business-logic/User";
+
+import { hashPassword } from "@helpers/auth";
+import prisma from "@helpers/prisma";
+import { teardown } from "@helpers/tests/teardown";
+
+let hashedPassword:string;
+let userEmail = "lambert@gmail.com";
+
+describe("User",()=>{
+    beforeEach(async () => {
+        await teardown();
+        hashedPassword = await hashPassword("lambert12345");
+      });
+    
+      describe('Users Test',()=>{
+          it('should create new user',async()=>{
+              const userBody = {
+                  name:"Nsengimana Lambert",
+                  email:userEmail,
+                  password:hashedPassword
+              }
+              const user = new UserEntity()
+              const userCreation = await user.create(userBody)
+              const { message } = userCreation
+              expect(message).toBe("User created")
+          });
+          it('should avoid creating user when exists',async()=>{
+            const userBody = {
+                name:"Nsengimana Lambert",
+                email:userEmail,
+                password:hashedPassword
+            }
+            const user = new UserEntity()
+                await user.create(userBody);
+                await expect(async () => {
+                await user.create(userBody);
+              }).rejects.toThrowError("Email address is already registered.");
+          })
+          it('should not create user when password is less than 7 characters',async()=>{
+            const userBody = {
+                name:"Nsengimana Lambert",
+                email:"lamb@gmail.com",
```

Comment:
can you use `...@example.com` addresses?

Reply:
Yes Please